### ISSUE TITLE
Why is META.json set at version 2.0.2 in master?

### ISSUE THREAD

**Issue by deinspanjer** [plv8/plv8#219]
Please forgive me if this is a dumb question, but I'm writing a PR that uses META.json to get the current version, and I noticed that in master, it is set to 2.0.2, but in the makefile it is 2.0.4-dev while the last tagged release, v2.0.3 has it set to 2.0.3.

What is the strategy for incrementing the version number in this file?

**Comment by JerrySievert** [comment]
`master` is the development branch - you'll find specific version in branches and tagged with specific version numbers.  these also correspond to the _releases_ link in GitHub.  you should not be using `master` for production unless you know exactly what you're doing and why you're doing it :)

`META.json` is used for PGXN.

is there some reason you can't use either `pg_extension` of the other methods plv8 exposes for versioning?

**Comment by deinspanjer** [comment]
Sorry, I should have provided more details than just "writing a PR".

We are using Postgres in Amazon AWS RDS, and the plv8 extension is available there, but it is at version 1.5.3 currently.

In our local developer sandboxes, we use Postgres inside a Docker container, and I ran into a small issue with the default version of plv8 that is available from Debian's apt repository being even older at 1.4.9.

Because of this, I've written a small Dockerfile and build script that can build plv8 inside a docker container and install the built version in a corresponding Postgres Docker image.  I would like to offer that as a contribution to your project.

The build script parses META.json to determine the default version of plv8 to retrieve and build.  It can be overridden, but I figured that would be a convenient way to keep it in sync with your tag and release mechanism.

While developing that, I noticed that master had a different version number in META.json than what I expected.
